tensorboard/plugins/custom_scalar/custom_scalars_plugin_test.py

```diff
@@ -92,6 +92,11 @@ def __init__(self, *args, **kwargs):
                                     upper='cross entropy upper'),
                             ],
                         )),
+                    layout_pb2.Chart(
```

Comment:
Maybe extend the comment on line 81 to say "and contains a duplicate chart also in that run" or something?